modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/higher_level_reviews_controller.rb

```diff
@@ -11,7 +11,7 @@ class AppealsApi::V2::DecisionReviews::HigherLevelReviewsController < AppealsApi
   include AppealsApi::Schemas
 
   skip_before_action :authenticate
```

Comment:
Since ICN is required to download a PDF, I used these existing `validate_index_headers` methods to check for it. I renamed them to make them sounds less specific to the index route, but I didn't make any other changes to them - I do think that they could move into a concern at some point (a concern for header validations maybe?)